freqtrade/freqtradebot.py

```diff
@@ -0,0 +1,535 @@
+"""
+Freqtrade is the main module of this bot. It contains the class Freqtrade()
+"""
+
+import copy
+import json
+import time
+import traceback
+from typing import Dict, List, Optional, Any, Callable
+from datetime import datetime
+import requests
+import arrow
+from cachetools import cached, TTLCache
+from freqtrade.analyze import Analyze
+from freqtrade.constants import Constants
+from freqtrade.fiat_convert import CryptoToFiatConverter
+from freqtrade.logger import Logger
+from freqtrade.persistence import Trade
+from freqtrade.rpc.rpc_manager import RPCManager
+from freqtrade.state import State
+from freqtrade import (DependencyException, OperationalException, exchange, persistence)
+
+
+class FreqtradeBot(object):
```

Comment:
I'm not 100% sure about that, but I think we should consider moving this class to `freqtrade/__init__.py`